server/series/models.py

```diff
@@ -21,19 +24,31 @@ class Category(models.TextChoices):
 
     start_date = models.DateField()
     end_date = models.DateField()
-    name = models.CharField(max_length=255)
+    name = models.CharField(max_length=100)
+    slug = models.SlugField(max_length=100, null=True, blank=True, db_index=True)
     type = models.CharField(max_length=3, choices=Type.choices)
-    catgory = models.CharField(max_length=7, choices=Category.choices)
+    category = models.CharField(max_length=7, choices=Category.choices)
     series_roster_max_players = models.PositiveSmallIntegerField()
     event_min_players_male = models.PositiveSmallIntegerField()
     event_min_players_female = models.PositiveSmallIntegerField()
     event_max_players_male = models.PositiveSmallIntegerField()
     event_max_players_female = models.PositiveSmallIntegerField()
     player_transfer_window_start_date = models.DateField(blank=True, null=True)
     player_transfer_window_end_date = models.DateField(blank=True, null=True)
-    season = models.ForeignKey(Season, on_delete=models.SET_NULL, null=True, blank=True)
+    season = models.ForeignKey(
+        Season, on_delete=models.SET_NULL, null=True, blank=True, related_name="series"
+    )
     teams = models.ManyToManyField(Team, related_name="series")
 
+    class Meta:
+        unique_together = ["season", "name"]
+
+    def save(self, *args: Any, **kwargs: Any) -> None:
+        if not self.slug:
+            self.slug = slugify(self.name)
```

Comment:
you might wanna generate unique slug by first checking if it exists etc.. like we do for event and team

Reply:
done